### PR TITLE
Add option 'ignoreError' in function extraction

### PR DESCRIPTION
**Pull request by Suhoy95**
#7

### CODE REVIEW

**Comment by Suhoy95** [comment]
don't know how make it good. Check this variant. 

If it's good i'll write tests.

File: src/builders.js [hunk 1]
```diff
@@ -124,31 +124,34 @@ module.exports = {
...
-      entry = buildPluralContextEntry(args, types, path, func)
-      return buildReference(entry, state)
+      func = getPluralContextFunction(state, callee)
+      if (func) {
+        entry = buildPluralContextEntry(args, types, path, func)
+        return buildReference(entry, state)
+      }
+    } catch (error) {
+      if (!func) throw error
+      if (func.validate) throw error
```
Comment (rtymchyk) [inline src/builders.js]: I'm not sure this works. Validation still runs regardless of whether func.validate is true or false.

**Comment by rtymchyk** [comment]
Looks like a good path forward

**Comment by rtymchyk** [comment]
Taking this over to get it over the finish line

**Pull request merged**